### PR TITLE
Add fixes for Nextcloud-related webapps

### PR DESCRIPTION
**Pull request by csolisr**
This commit fixes the parsing algorithm for the website to use `safeParseInt` across all Nextcloud applications, modifies the default items on the settings dropdown menu to be better localized to each individual page, and corrects the format of the URL to better accommodate the custom servers used by Nextcloud.

### CODE REVIEW

File: recipes/nextcloud-talk/webview.js [hunk 1]
```diff
@@ -5,21 +5,24 @@ function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { de
...
     let direct = 0;
+
     const notificationWrapper = document.querySelector(
       '.notifications .notification-wrapper',
     );
 
     if (notificationWrapper) {
-      direct = notificationWrapper.querySelectorAll(
+      const directSelector = notificationWrapper.querySelectorAll(
         '.notification[object_type="chat"], .notification[object_type="room"',
```
Comment (csolisr) [inline recipes/nextcloud-talk/webview.js:15]: Please wait until PR #64 is merged. This line needs to be corrected accordingly.

**Comment by vraravam** [comment]
@csolisr - please go ahead, I have merged #64

File: recipes/nextcloud-carnet/package.json [hunk 2]
```diff
@@ -1,13 +1,15 @@
 {
   "id": "nextcloud-carnet",
   "name": "Carnet (Nextcloud)",
-  "version": "1.2.0",
+  "version": "1.3.0",
```
Comment (SpecialAro) [inline recipes/nextcloud-carnet/package.json:4]: The version bump should be `1.2.1` and not `1.3.0`

File: recipes/nextcloud-talk/webview.js (hunk 1 above)
Comment (csolisr) [reply recipes/nextcloud-talk/webview.js]: Merge solved

File: recipes/nextcloud-carnet/package.json (hunk 2 above)
Comment (csolisr) [reply recipes/nextcloud-carnet/package.json]: Thanks for the clarification, done

**Review by vraravam — CHANGES_REQUESTED** [review]

File: recipes/nextcloud-talk/package.json [hunk 3]
```diff
@@ -3,10 +3,13 @@
   "name": "Nextcloud Talk",
   "version": "1.3.1",
```
[the remark this answers is not in the document]
Comment (csolisr) [reply recipes/nextcloud-talk/package.json]: Done, please review

**Review by vraravam — APPROVED** [review]

**Comment by vraravam** [comment]
@all-contributors please add @csolisr for code

**Pull request merged**